crates/rust-analyzer/src/code_lens_resolve_data.rs

```diff
@@ -0,0 +1,10 @@
+//! Information about CodeLens, that is to be resolved.
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) enum CodeLensResolveData {
```

Comment:
Is there an existing module where `CodeLensResolveData` can be moved to?

Reply:
lsp_ext.rs would be the right place.